pettingzoo/test/example_envs/generated_agents_env_v0.py

```diff
@@ -75,12 +75,25 @@ def reset(self, seed=None, options=None):
         self.truncations = {}
         self.infos = {}
         self.num_steps = 0
+
+        self._obs_spaces = {}
+        self._act_spaces = {}
+        self.types = []
+        self._agent_counters = {}
+        for i in range(3):
+            self.add_type()
```

Comment:
What's going on here?

Reply:
This is used in the init but wasn’t in the reset(), very bizarre code but essentially adds randomized obs and action spaces. Problem was that calling reset(seed) wasn’t making things consistent.